app/views/editions/secondary_nav_tabs/_edit.html.erb

```diff
@@ -0,0 +1,127 @@
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <%= render "govuk_publishing_components/components/heading", {
+      text: "Edit",
+      heading_level: 2,
+      margin_bottom: 5,
+    } %>
+  </div>
+</div>
+
+<%= resource_form(@resource) do |f| %>
```

Comment:
Should we be using `resource_form`? It appears to be a helper method that adds some "data-module" attributes to do with AJAX-saving.